src/string-class.js

```diff
@@ -0,0 +1,93 @@
+// Disable eslint do not modify native methods
+/* eslint-disable no-extend-native */
+
+String.prototype.hasVowels = function hasVowels() {
+  // Returns true if a vowel exists in the input string. Returns false otherwise.
+  const vowels = new RegExp('[aeiou]', 'i');
+  return vowels.test(this);
+};
+
+String.prototype.toUpper = function toUpper() {
+  /* Replace all lowercase letters in the input string with their uppercase
+  * analogs by converting each letter's ASCII value to decimal then back to
+  * ASCII
+  */
+  const upper = new RegExp('[a-z]', 'g');
```

Comment:
No need to use the `RegExp` constructor. The literal syntax would be good enough for this case.